.travis.yml

```diff
@@ -28,3 +28,4 @@ script: chmod +x ./.travis/build.sh && ./.travis/build.sh
 jdk:
   - oraclejdk8
   - oraclejdk9
+  - oraclejdk10
```

Comment:
Let's remove or comment out oraclejdk10 because it is not available in travis yet.